app/src/main/java/uk/ac/ebi/atlas/experimentpage/JsonExperimentMetadataController.java

```diff
@@ -38,7 +42,7 @@ public String getTSnePlotMetadata(@PathVariable String experimentAccession,
 
         return GSON.toJson(
                 ImmutableMap.of(
-                        "perplexities", experimentPageContentService.getPerplexities(experiment.getAccession()),
+                        "plotTypesAndOptions", GSON.toJsonTree(tsnePlotSettingsService.getAvailablePlotTypesAndPlotOptions(experimentAccession)),
                         "metadata", experimentPageContentService.getMetadata(experiment.getAccession()),
```

Comment:
get `accession` from the experiment like the previous code base.

Reply:
Did you try running the WIT test after this change?

Reply:
We don't have a WIT for this controller.